test-cases/nemesis/longevity-10gb-3h-AbortRepairMonkey.yaml

```diff
@@ -0,0 +1,24 @@
+test_duration: 255
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=180m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=1000 -pop seq=1..10000000 -log interval=5"
```

Comment:
In continue to my previous comment, I want to have a new yaml of minimal longevity, here are my suggested changes:


Change the stress to be the same as in 100gb-4h, but with 5GB and 1h.